rustls/examples/internal/bogo_shim_impl.rs

```diff
@@ -13,12 +13,22 @@ use base64::prelude::{Engine, BASE64_STANDARD};
 use pki_types::{CertificateDer, PrivateKeyDer, ServerName, UnixTime};
 use rustls::client::danger::{HandshakeSignatureValid, ServerCertVerified, ServerCertVerifier};
 use rustls::client::{ClientConfig, ClientConnection, Resumption, WebPkiServerVerifier};
-#[cfg(all(not(feature = "ring"), feature = "aws_lc_rs"))]
+#[cfg(feature = "aws_lc_rs")]
+use rustls::client::{EchConfig, EchGreaseConfig, EchMode, EchStatus};
```

Comment:
I don't mind either way, but I would write a big compound use here to avoid so many feature gates.